uefi/src/data_types/strs.rs

```diff
@@ -423,6 +423,12 @@ impl<StrType: AsRef<str> + ?Sized> EqStrUntilNul<StrType> for CStr16 {
     }
 }
 
+impl AsRef<CStr16> for CStr16 {
```

Comment:
I thought that there is a default implementation for that by the core lib. Apparently, there is not.